src/map.rs

```diff
@@ -1126,6 +1194,20 @@ where
         self.count.load(Ordering::Relaxed)
     }
 
+    #[inline]
+    /// Returns the capacity of the map.
+    pub fn capacity<'g>(&self, guard: &'g Guard) -> usize {
```

Comment:
I'm actually not sure we want to expose this method publicly. There (probably) aren't any good reasons why callers should care how many bins there were, especially given concurrency, so I think I'd just make this private. And maybe even `#[cfg(test)]`.

Reply:
I just added it for now to be able to write the tests and to get your opinion on this. I think `#[cfg(test)]` would be the most reasonable

Reply:
it's not actually that easy as just slapping a `#[cfg(test)]` on there, since the tests in `tests` don't trigger that. (I assume, at least cargo check complained about not being able to find `capacity`).
I'll see if I can sort this out tomorrow

Reply:
Ah, yes, you're right.. That's awkward. I guess leave it as `pub` then :man_shrugging:

Reply:
I just looked at this a bit more and if we make tests that use `capacity` unit tests (so move them to the same module) It'll work, so I'll do that.